bears/general/InvalidLinkBear.py

```diff
@@ -94,7 +94,8 @@ def run(self, filename, file,
                         line=line_number,
                         severity=RESULT_SEVERITY.NORMAL)
                 if 300 <= code < 400:  # HTTP status 30x
-                    redirect_url = requests.head(link, allow_redirects=True).url
+                    redirect_url = requests.head(
+                        link, allow_redirects=True).url
```

Comment:
I am not sure aligning column wise would exceed the limit here? If it doesn't then maybe we should prefer doing it.